oslabs-beta/Kubiquity
# Connect FE to BE for /metrics

WHAT DOES THIS PR DO? 
1. Establishes naive connection between FE and BE for `/metrics` route. 
2. Updates fake metrics data. 
3. Adds labels to BarChart. 
4. Rename `Pods` component to `Metrics` component. 

HOW TO TEST THIS PR:
0. EDIT: Do not test this PR until https://github.com/oslabs-beta/Kubiquity/pull/19 has been merged in. 
1. Make sure you have the most recent backend work merged in, as you will need it to test this PR. `gco main && git pull upstream main`
2. Run `gh pr checkout 18 && npm install && npm run dev`. 
3. Once the Electron app has fully loaded, verify that the Metrics data now reflects the metric schema: `{ podId: Number, memory: Number }`
4. Verify that the two fake metrics load in the graph, and that the graph has the correct labels. 
5. In the `Errors` component, uncomment line 17 and comment out line 18. 
6. In the `Metrics` component, comment out line 27 and comment in lines 30-33. 
7. Follow points 2–8 on https://github.com/oslabs-beta/Kubiquity/pull/17 to get your containers running and your cluster spun up. 
8. Once everything has loaded and connected (cough mongo cough), verify that the app is being populated with actual data instead of mock / test data. 

ADDITIONAL NOTES: 
- 🙏
